static-catalog/examples/CBERS/CBERS_4_MUX_20170528_090_084_L2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": "CBERS_4_MUX_20170528_090_084_L2", 
+  "type": "Feature", 
+  "bbox": [
+    13.700498, 
+    23.465111, 
+    14.98818, 
+    24.812825
+  ], 
+  "geometry": {
+    "type": "MultiPolygon", 
+    "coordinates": [
+      [
+        [
+          [
+            13.891487, 
+            23.463987
+          ], 
+          [
+            13.723733, 
+            24.567134
+          ], 
+          [
+            14.786396, 
+            24.811268
+          ], 
+          [
+            14.95489, 
+            23.703021
+          ], 
+          [
+            13.891487, 
+            23.463987
+          ]
+        ]
+      ]
+    ]
+  }, 
+  "properties": {
+    "datetime": "2017-05-28T09:01:17Z", 
+    "provider": "INPE", 
+    "license": "CC-BY-SA-3.0"
+  }
+}
```

Comment:
Looks like you don't have the 'assets' yet, but putting the full metadata file in the asset list would be a good idea. Then people can get the ephemeris and other more obscure metadata.

Reply:
Just included the 'eo' section based on the current Landsat-8 and Planet examples. 

Is it possible to assume that all angles (sun_elevation, sun_azimuth, off_nadir_angle) are in degrees and distances (resolution) are in meters?

For eo:off_nadir_angle, is there a convention about the angle sign or are we interested here only in the absolute value?